lib/util/src/arch.c

```diff
@@ -12,6 +12,9 @@
 
 #if __amd64__
 
+/* Use noinline to try to prevent -flto from inlining assembly.
+ */
+__attribute__((__noinline__))
```

Comment:
@tristan957 I tried adding support for this attribute, but was unable to figure out how to get it generated in the build_config.h file...

Reply:
Greg, I can just take a look in follow-up PR to get the CI for this going if you want.

Reply:
Yes please, thanks!

Reply:
Cool. Thank you for this description. Will take it into account in follow-up.